scss/upload/_theme.scss

```diff
@@ -132,4 +132,27 @@
     .k-dropzone-hovered {
         background-color: $hovered-bg;
     }
+
+    // Metrics
+    .k-dropzone {
+        .k-upload-status {
+            >.k-icon {
+                margin-right: $padding-x-lg / 2;
```

Comment:
We have `$icon-spacing`, `$spacer` and `$spacer-x` variables that could be used.